microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -1,70 +1,91 @@
+
+When Microsoft Edge sends an HTTPS request to a server, it sends a set of [low entropy][https://wicg.github.io/client-hints-infrastructure/#low-entropy-table] User-Agent Client Hints headers. If the server requires more granular information about the browser, its response includes an `Accept-CH` header. The value of that response header is a comma-separated list of all the Client Hints request headers the server wants from the browser, such as `Accept-CH: Sec-CH-UA-Full-Version,Sec-CH-UA-Platform-Version`. The next Microsoft Edge HTTPS request to the server will include the specified User-Agent Client Hints headers.
+
+By default, Microsoft Edge sends the `Sec-CH-UA`, `Sec-CH-UA-Mobile`, and `Sec-CH-UA-Platform` request headers in the following format.  
 
 ```https
-Sec-CH-UA: "Chromium";v="91", "Microsoft Edge";v="91","Dummy;Browser Brand";v="99"
+Sec-CH-UA: "Chromium";v="92", "Microsoft Edge";v="92","Placeholder;Browser Brand";v="99"
 Sec-CH-UA-Mobile: ?0
+Sec-CH-UA-Platform: "Windows"
 ```  
 
-> [!NOTE]
-> Client Hints are only sent over secure connections using `HTTPS`.  
+The following table shows all available hints request headers with sample values.
 
-The following low entropy hints are sent by default.  If you need to access more information, send one of the following request headers.  
-
-#### User-Agent request and response headers  
-
-| Request header | Example response value |  
+| User-Agent request header | Example User-Agent response value |  
 |:--- |:--- |  
 | `Sec-CH-UA` | `"Chromium";v="91", "Microsoft Edge";v="91","GREASE";v="99"` |  
-| `Sec-CH-UA-Mobile` | `false` |  
+| `Sec-CH-UA-Mobile` | `?0` |  
 | `Sec-CH-UA-Full-Version` | `91.0.866.0` |  
 | `Sec-CH-UA-Platform` | `Windows` |  
-| `Sec-CH-UA Platform-Version` | `10.0` |  
-| `Sec-CH-UA-Arch` | `x86` |  
-| `Sec-CH-UA-Model` |  |  
+| `Sec-CH-UA-Platform-Version` | `10.0` |  
+| `Sec-CH-UA-Arch` | `x86` | 
+| `Sec-CH-UA-Bitness` | `64` |
+| `Sec-CH-UA-Model` | `Surface Pro` |  
+
+> [!NOTE]
+> User-Agent Client Hints are only sent over secure connections using `HTTPS`.
 
 ### User-Agent Client Hints JavaScript API  
 
-The default response value from `navigator.userAgentData` uses the following format.  
+You can access User-Agent Client Hints using JavaScript on the client side. When you call the default `navigator.userAgentData` it returns the following response.  
 
 ```javascript
 { brands: [ {brand: "Chromium","version":"91"}, {brand: "Microsoft Edge","version":"91"}, {brand: "GREASE","version":"99"}, ]
 mobile: false }
 ```  
 
-If you need to access more detailed information, use the [getHighEntropyValues()][GithubWicgUaClientHintsGethighentropyvalues] method.  
+Microsoft Edge includes a `GREASE` brand value that changes over time. It prevents sites from matching the entire brand list when attempting to detect a version of Microsoft Edge.
+
+To request more detailed information such as `platform`, use the following code.
 
 :::row:::

```

Comment:
Can we remove the side-by-side format? Let's stack the code blocks vertically instead?

Reply:
done